az-hadoop-jobtype-plugin/src/main/java/azkaban/jobtype/HadoopJavaJobRunnerMain.java

```diff
@@ -94,6 +94,7 @@ public void run() {
     // Separate try catch for logger
     try {
       _logger = Logger.getRootLogger();
+      _logger.setLevel(Level.INFO);
```

Comment:
Do we want to set this from configuration such that it can be changed without a code change? We have a lot of things in configs already, not sure if it would be useful to add this as well, but thought I'd ask the question.

Reply:
Plus one. This should come from configuration.

Reply:
Agreed to perhaps address this in a future change as this is blocking and our logging strategy might change in the near future anyway, rendering this obsolete.